src/Altinn.Platform/Altinn.Platform.Authorization/IntegrationTests/Util/AssertionUtil.cs

```diff
@@ -202,6 +202,24 @@ public static void AssertResourcePolicyResponseEqual(ResourcePolicyResponse expe
             }
         }
 
+        /// <summary>
+        /// Assert that two <see cref="ResourcePolicyResponse"/> have the same property in the same positions.
```

Comment:
cref seems to be copied from another method

Reply:
fixed this comment